lang/cs/Org.Apache.REEF.Network.Examples/GroupCommunication/BroadcastReduceDriverAndTasks/BroadcastReduceDriver.cs

```diff
@@ -19,10 +19,8 @@
 using System.Collections.Generic;
```

Comment:
Why do we need to move Comm. group creation out of constructor here...For IIMRU Driver we needs to introduce FT and hence those changes are necessary. Here we can leave them as they are. The problem I foresee is that ContextManager would need to be modified and made specific to IMRU in future and hence I feel we should let this piece of code as it is in term of organization ant not use ContextManager here.